common/djangoapps/util/views.py

```diff
@@ -434,6 +440,12 @@ def build_error_response(status_code, field, err_msg):
 
     success = False
     context = get_feedback_form_context(request)
+
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
```

Comment:
@brittneyexline @douglashall Today, I discussed one thing with @saleem-latif in order to identify the default behavior of `fetch_enterprise_learner_data` in case of Anonymous users or User with empty string. We identified that API would return **all users data** if the given username is an empty string. 
So may be we need to update the [fetch_enterprise_learner_data](https://github.com/edx/edx-platform/blob/master/openedx/features/enterprise_support/api.py#L87
) method to handle this case. We are using this api in many other places as well.
But I am not sure, Would be there any other scenario that Derek has mentioned on JIRA.